docs/common/yum.md

```diff
@@ -91,6 +91,25 @@ getting the OSG-supported versions.
         :::file
         plugins=1
 
+### Enable the "extras" OS repositories
+
+Some packages depend on packages in the "extras" repositories of your distro,
```

Comment:
Since we call it an OS above, let's keep going with that

```suggestion
Some packages depend on packages in the "extras" repositories of your OS,
```